cookbook/Basic Prompt Routing/README.md

```diff
@@ -0,0 +1,12 @@
+# Basic Prompting Routing Demo - AI Teaching Assistant
```

Comment:
Hmm I would recommend changing the name of the folder to be a single word. Maybe `Basic-Prompt-Routing`?